spec/lib/hexagram/repositories/repository_spec.rb

```diff
@@ -111,19 +111,55 @@
     it "returns an empty instance of a CampusEntity" do
       rand_value = rand
       rand_id = rand(5)
-      persistence_class = Persisters::Test
       persistence_instance = Persisters::Test.new
       persistence_instance.id = rand_id
       persistence_instance.value = rand_value
 
       allow(persistence_class).to receive(:find).with(rand_id).and_return(persistence_instance)
 
-      repository = described_class.new(persistence_class, orm_adapter)
-
       ret = repository.find(rand_id)
       expect(ret).to be_a(Entities::Test)
       expect(ret.value).to eq(rand_value)
       expect(ret.id).to eq(rand_id)
     end
   end
+
+  describe "all" do
+    before do
+      @rand_value = rand
```

Comment:
I tried to use let instead, but I get the following error:

`let` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).